### PR TITLE
Introducing cover letter draft and action

### PR DESCRIPTION
**Pull request by pedrohbraga**
Hi,

In this pull request, I include a Github Action that renders an RMarkdown containing editorial communication letters to HTML, and that then pushes it to a path in the `gh-pages` branch.

The original Manubot action will still run when we make changes to the `editorial-communication` path, but the action that I am running here will not run if we do not push changes directly to this path. Making Manubot ignore this path seems a bit complicated, but I do not expect this to interfere with this process.

As soon as I can confirm that this action is working, I will sweep and clean any unnecessary files (e.g. the `.bib` one).

Let us try and see how it works!

I will make sure to keep an eye out to see if everything is working fine.

### CODE REVIEW

**Comment by pedrohbraga** [comment]
It looks like the Github Action worked!

The letter webpage is here: https://sortee-github-hackathon.github.io/manuscript/editorial-communication/cover-letter-submission.html

It is a very simple page!

Let me know what you think and if you would like something else to be added!

File: editorial-communication/cover-letter-submission.Rmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+knitr::opts_chunk$set(fig.retina = 2)
+```
+Dear Editor,
+
+We would like to submit our manuscript entitled “Not just for programmers: A friendly guide on the versatility of GitHub for accelerating collaborative research in Ecology and Evolution” for consideration to be published in OUR_CHOICE_OF_JOURNAL. 
+Our study provides researchers in ecology and evolution with a critical review on thirteen practical ways they can use Github -- an online, cloud-based service for cooperative and documented code development -- to make their research more collaborative, transparent, and reproducible.
+
+Scientific advancement fundamentally requires researchers to adopt reproducible, collaborative and transparent practices.
+
+Most scientists, including ecologists and evolutionary biologists, have become critically dependent on computational tools in their research.
```
Comment (LunaSare) [inline editorial-communication/cover-letter-submission.Rmd:21]: Maybe a citation here?

**Review by LunaSare — APPROVED** [review]
It looks great @pedrohbraga! 
I think it has all the necessary elements of a cover letter, including some compelling arguments! 
I sometimes include references in my cover letters, and as I was reading this one, I felt I would have added at least one on L 21 of [editorial-communication/cover-letter-submission.Rmd](https://github.com/SORTEE-Github-Hackathon/manuscript/pull/170/files#diff-8cd669dac58a5388eb3cf564c21b235efdb8857adc26854ddbf0b32adc3b865c).

But let's see what others think.

Thank you for your work, @pedrohbraga!

**Review by Aariq — APPROVED** [review]

File: editorial-communication/cover-letter-submission.Rmd (hunk 1 above)
Comment (robcrystalornelas) [reply editorial-communication/cover-letter-submission.Rmd:21]: Yes, I agree with adding a couple citations to the cover letter to show that we know the state of current GitHub use in EcoEvo

**Review by robcrystalornelas — APPROVED** [review]
Great @pedrohbraga. I added in a paragraph that includes some implications. We can absorb this into other paragraphs if that's better too. The journal also asks us to indicate potential reviewers so I included space for that.

**Review by Aguncan — APPROVED** [review]
very good

**Pull request merged**